### DISCUSSION TITLE
spatie/laravel-event-sourcing

### DISCUSSION THREAD

**Discussion by jnye** [HassanZahirnia/laravel-package-ocean#60] [Package Suggestions]
This package aims to be the entry point to get started with event sourcing in Laravel. It can help you with setting up aggregates, projectors, and reactors.

Repo: https://github.com/spatie/laravel-event-sourcing

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package! It will be live in a moment.
